shared/attributes.asciidoc

```diff
@@ -430,7 +430,7 @@ Legacy definitions
 ///////
 Temporarily reroute APM Agent links to the new APM documentation book
 ///////
-ifeval::["{branch}"=="7.16"]
+ifeval::["{branch}"=="7.17"]
```

Comment:
Thanks for catching this. We should be able to remove this `if` block as we didn't have time to fix all of the broken links in 7.16, but did for 7.17. I'll wait until after the release to remove it though—just in case 😅.